src/main/kotlin/de/amplimind/codingchallenge/controller/AdminController.kt

```diff
@@ -45,12 +39,22 @@ class AdminController(
     @ApiResponse(responseCode = "404", description = "User with email was not found.")
     @GetMapping("fetch/projects/{email}")
     fun fetchUserInfosForEmail(
-        @PathVariable email: String,
+            @PathVariable email: String,
     ): ResponseEntity<UserInfoDTO> {
         val userInfo = this.userService.fetchUserInfosForEmail(email)
         return ResponseEntity.ok(userInfo)
     }
 
+    @Operation(summary = "Endpoint for deleting a user by email")
+    @ApiResponse(responseCode = "200", description = "User was deleted successfully.")
+    @DeleteMapping("user/{email}")
+    fun deleteUserByEmail(
+            @PathVariable email: String,
+    ): ResponseEntity<UserInfoDTO> {
+        this.userService.deleteUserByEmail(email)
```

Comment:
Wir brauchen am besten auch noch einen check in der service methode dass man sich nicht selbst deleten kann^^
Also dass sich der User der diesen request macht nicht der sein darf welcher gelöscht werden soll.
Wer gerade eingeloggt ist kriegst du über (musst dann vllt noch zu nen User casten):

![image](https://github.com/amplimindcc/backend/assets/68387530/fffdc5c9-1f27-46de-8769-4701177fef1b)

Reply:
Wenn man sich selbst deleten will dann am besten einen HttpStatus Conflict zurück geben^^

Reply:
Und auf die Aufgabe (Submission) zu einen User der deleted werden soll muss deleted werden